modules/apps/app-builder/app-builder-web/test/js/utils/normalizers.es.js

```diff
@@ -215,9 +215,6 @@ describe('normalizers', () => {
 		expect(
 			normalizedDataLayout.dataLayoutFields.Text97181041
 		).toStrictEqual({
-			label: {
```

Comment:
This label were removed, because is no longer needed from fieldProperties @interaminense